### DISCUSSION TITLE
All sensors showing Unknown

### DISCUSSION THREAD

**Discussion by Snypes88** [hcpy2-0/hcpy#100] [Q&A] [answered]
Hello, I have a Gaggenau Steam Oven and I was able to succesfully set everything up in a docker container, however, I am getting Unknown values on all my HA sensors. Can someone point me in the right direction?

![image](https://github.com/user-attachments/assets/fc84d5ff-4d82-4ce3-813d-802c987d8735)

This is a screenshot of my terminal output from my docker container (running on a raspberry pi)

![image](https://github.com/user-attachments/assets/593b9d4b-6870-44da-a304-2e33a629579b)

**Comment by Meatballs1** [comment]
Logs  would be the best place to start:

`docker-compose -f compose.yaml logs`

**Comment by Meatballs1** [comment, marked as answer]
So it looks like DNS cant be resolved for the device. In `devices.json`  the `host` field contains the address it connects to.  Check you can resolve that from where its executing. Consider adding your local suffix or changing to an IP (but you will probably want to ensure the IP is static or fixed in DHCP).

**Reply by Snypes88** [reply]
That seems to have been it! Thanks for the super fast response 🙏 I have a pihole running on my network and reverse lookup has never been working, still struggling with it so I just added a line to /etc/hosts with the ip address and host name so that I can maintain the devices.json as is. Thanks again for the lightning fast diagnosis!
